src/fixture/fixture.js

```diff
@@ -0,0 +1,64 @@
+import { nextFrame, fixture as wcFixture } from '@open-wc/testing';
+import { reset } from './reset.js';
+
+function getComposedChildren(node) {
```

Comment:
The original draft version used this method from `@brightspace-ui/core`. I don't love this but I had to copy it into here since core is going to eventually use this library and we can't have circular dependencies. If anyone has a better idea...

Reply:
Thanks, I'll at least try simplifying this to only include the stuff we care about.

Agreed that handling Polymer likely doesn't make any sense, but if we did need to that future plan to `await` a `getLoadingComplete()` Promise could work in a pinch.

Reply:
I was able to simplify this a bit. Because of the way this encounters shadow roots, this function was never called with a shadow root as the root element, so checking for `ELEMENT_NODE` was sufficient.